src/main/resources/db/migration/V1_29__insert_sscs_record.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM judicial_service_code_mapping WHERE service_id ='21' OR service_id ='55';
```

Comment:
Please use the following query for delete:

DELETE FROM judicial_service_code_mapping WHERE service_id in ('21', '55');
DELETE FROM judicial_ticket_code_mapping WHERE ticket_code  in ('363', '376');